client/src/components/App/Content/LandingPage/InvestigationTable/FilterCreators.ts

```diff
@@ -132,6 +132,16 @@ export const filterCreators: { [T in InvestigationsFilterByFields]: ((values: an
                 }
               }
         } } :  { [InvestigationsFilterByFields.UNUSUAL_COMPLETED_NO_CONTACT]: null };
+    },
+    [InvestigationsFilterByFields.UNALLOCATED_DESK]: (isFilterOn: boolean) => {
+        return isFilterOn ?
+            {
```

Comment:
I think you don't need this {}

Reply:
[InvestigationsFilterByFields.UNALLOCATED_DESK]: (isFilterOn: boolean) => {
        return isFilterOn ? [InvestigationsFilterByFields.UNALLOCATED_DESK]: {deskId: { isNull: true } }
            :[InvestigationsFilterByFields.UNALLOCATED_DESK]: null;

Reply:
I need the curly brackets because the function returns an object